core/src/main/java/com/neemre/btcdcli4j/core/client/BtcdClientImpl.java

```diff
@@ -166,16 +166,14 @@ public String createRawTransaction(List<OutputOverview> outputs,
 		return hexTransaction;
 	}
 
-	@Override
-	public RawTransactionOverview decodeRawTransaction(String hexTransaction)
-			throws BitcoindException, CommunicationException {
-		return decodeRawTransaction(hexTransaction, true);
-	}
-
 	@Override
 	public RawTransactionOverview decodeRawTransaction(String hexTransaction, Boolean isWitness)
 			throws BitcoindException, CommunicationException {
-		List<Object> params = CollectionUtils.asList(hexTransaction, isWitness);
+		List<Object> params = new ArrayList<>();
+		params.add(hexTransaction);
```

Comment:
If this is going to be called a lot, it would be worth it to do `new ArrayList<>(2)`. The default allocates enough spaces for 16 items, and you only need 2.

May seem like a pointless optimization but cloud computing makes performance and memory usage important factors cost-wise.

Reply:
Updated, thx!